app/models/generic_object_definition.rb

```diff
@@ -49,6 +45,24 @@ def property_defined?(attr)
     property_attribute_defined?(attr) || property_association_defined?(attr) || property_method_defined?(attr)
   end
 
+  def create_object(options)
+    GenericObject.create!({:generic_object_definition => self}.merge(options))
+  end
+
+  def find_objects(options)
```

Comment:
@lfu can you add comments here what fields are searchable, and possibly what are expected to be searchable in the future?